docs/sources/open-source.md

```diff
@@ -8,42 +8,64 @@ title: Open Source
 weight: 300
 ---
 
-# Open Source
+# Grafana OnCall Open Source guide
 
-We prepared three environments for OSS users:
-- **Hobby** environment for local usage & playing around: [README.md](https://github.com/grafana/oncall#getting-started).
+Grafana OnCall is a developer-friendly incident response tool that's available to Grafana open source and Grafana Cloud users. The OSS version of Grafana OnCall provides the same reliable on-call management solution along with the flexibility of a self-managed environment. 
+
+This guide describes the necessary installation and configuration steps needed to configure OSS Grafana OnCall. 
+
+The intended audience for this guide includes:
+- Grafana open source admins who are responsible for deploying and configuring Grafana OnCall.
+- Grafana open source users who need to configure SMS and phone notifications using Grafana Cloud.
+
+## Install Grafana OnCall OSS
+
+There are three Grafana OnCall OSS environments available:
+
+- **Hobby** playground environment for local usage: [README.md](https://github.com/grafana/oncall#getting-started)
 - **Development** environment for contributors: [DEVELOPER.md](https://github.com/grafana/oncall/blob/dev/DEVELOPER.md)
-- **Production** environment for reliable cloud installation using Helm: [Production Environment](#production-environment)
+- **Production** environment for reliable cloud installation: [Production Environment](#production-environment)
+
+For detailed installation instructions and additional resources, refer to the OSS Grafana OnCall [README.md](https://github.com/grafana/oncall#getting-started)
+
+For more information on production environment installation, refer to the following OSS Grafana OnCall [production environment helm chart](https://github.com/grafana/oncall/helm)
+
+
+## Slack integration for Grafana OnCall OSS
```

Comment:
Note that you'll want to edit the text that links to this topic. It looks like it hasn't been changed in Install Grafana OnCall OSS